lib/ChangeTypes.js

```diff
@@ -80,6 +80,8 @@ module.exports = {
   SIMPLIFY_FRACTION: 'SIMPLIFY_FRACTION',
   // e.g. 2/-3 -> -2/3
   SIMPLIFY_SIGNS: 'SIMPLIFY_SIGNS',
+  // e.g. 15/6 -> (5*3)/(2*3)
+  FIND_GCD: 'FIND_GCD',
```

Comment:
in your test (and the code you wrote) you'll get `15/3 / (6/3)` and not `(5*3)/(2*3)`

I think the latter is cleaner (nested fractions render more funky in the app) - what do you think is better?